app/components/listing/ResourceInfos.jsx

```diff
@@ -131,96 +104,30 @@ PhoneNumber.propTypes = {
 };
 
 
-function ExternalLink(props) {
-  return <a href={props.to} target="_blank" rel="noopener noreferrer">{props.children}</a>;
-}
+const ExternalLink = ({ children, to }) => <a href={to} target="_blank" rel="noopener noreferrer">{children}</a>;
```

Comment:
Nitpick: this line seems a good bit longer than the rest. I feel like it might be good to give this one the paren-wrapped multi-line treatment for readability.

Reply:
Agreed, I can make this change.